test/kokoro/xds.sh

```diff
@@ -1,15 +1,13 @@
 #!/bin/bash
 
 set -exu -o pipefail
-if [[ -f /VERSION ]]; then
-  cat /VERSION
-fi
+[[ -f /VERSION ]] && cat /VERSION
 
 cd github
 
-export GOPATH=${HOME}/gopath
-pushd grpc-go/interop/xds/client
-go build
+export GOPATH="${HOME}/gopath"
+pushd grpc-go
+go build ./interop/xds/client
```

Comment:
I believe go build always put the executable in the current directory.

So with this, client will be in `grpc-go/client`, not `grpc-go/interop/xds/client/client`. The following command won't find it.

Reply:
And I think what you had here before is OK.

Reply:
You're right. Switched back.

Reply:
Agreed - it was fine before, I was somewhat hopeful that you could do it without the pushd/popd at all.  (IMO Go's module behavior is wrong and it should scan starting at the build target's directory, not the current directory.)